app/policies/checklist/task_template_policy.rb

```diff
@@ -0,0 +1,12 @@
+module Checklist
```

Comment:
So this is the equivalent of
```
module Checklist
  module Organizations
    class TaskTemplatePolicy
```

Let's stick to nested modules throughout https://thoughtbot.com/blog/why-you-should-nest-modules-in-ruby

Further, I think we would want it to look like? @mononoken @jmilljr24 
```
module Organizations
  module Checklist
    class TaskTemplatePolicy
```

Reply:
@kasugaijin for clarification

this is what I currently have 

```
module Checklist
  class Organizations::TaskTemplatePolicy < ApplicationPolicy
    pre_check :verify_organization!
    pre_check :verify_active_staff!

    alias_rule :new?, :create?, :index?, to: :manage?

    def manage?
      permission?(:manage_task_templates)
    end
  end
end
```

but you would like it to be like the below and ensuing the checklist module to live under Organizations 

```
module Organizations
  module Checklist
    class TaskTemplatePolicy < ApplicationPolicy
      pre_check :verify_organization!
      pre_check :verify_active_staff!

      alias_rule :new?, :create?, :index?, to: :manage?

      def manage?
        permission?(:manage_task_templates)
      end
    end
  end
end
```

Reply:
Yes I think an organization would have a checklist rather than the other way round?